code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -22,15 +22,16 @@
 	med_hud_set_status()
 	handle_hud_icons_health()
 
-/mob/living/carbon/human/adjustBrainLoss(amount, updating = TRUE)
+/mob/living/carbon/human/adjustBrainLoss(amount, updating = TRUE, ignore_brain_mod = FALSE)

```

Comment:
minor style thing: Try to have variables named positively whenever you can, and tweak their values as necessary instead.
To illustrate, `ignore_brain_mod = FALSE` is the same as `avoid_not_ignoring_brain_mod = FALSE`, but figuring out what does `if(!avoid_not_ignoring_brain_mod)` means takes an unreasonable amount of time.
Similarly, it's much more straightforward to reason about `if(use_brain_mod)` than about `if(!ignore_brain_mod)`.

Reply:
Using positive variables (do_x) is usually more readable than negative variables(do_not_do_x).
Negative does not always exclude what it does do example. do_not_harm, what will it do then? Heal?

Reply:
I like use_brain_mod instead